lib/how.md

```diff
@@ -0,0 +1,35 @@
+## Config
```

Comment:
Trivial, but make this a `README.md` file, so that it shows up when you browse to this directory in GitHub.